src/pheval/runners/runner.py

```diff
@@ -14,6 +22,91 @@ class PhEvalRunner(ABC):
     output_dir: Path
     config_file: Path
     version: str
+    directory_path = None
+
+    def _get_tool(self):
+        return parse_input_dir_config(self.input_dir).tool
```

Comment:
From the parse_input_dir_config method, it seems the parameter it expects the path to a config file, not a directory.

Reply:
Yes, it expects the config file for the input dir, I thought this would be the input now for the `--input-dir` parameter? If not I can set it back to the directory and specify the file name from there to read

Reply:
Ah wait yes, I think I misunderstood from last week, a directory is still expected as the parameter for the `--input-dir` but as a developer, you can also expect a `config.yaml` within that directory that you can access?

Reply:
> Ah wait yes, I think I misunderstood from last week, a directory is still expected as the parameter for the --input-dir but as a developer, you can also expect a config.yaml within that directory that you can access?

Exactly!

Reply:
Understood, will make those changes back to expecting a directory